week08_pomdp/practice_tensorflow.ipynb

```diff
@@ -480,42 +580,50 @@
-      "text/plain": [
-       "<matplotlib.figure.Figure at 0x7f14cbf28860>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "  6%|▌         | 584/10000 [02:24<38:50,  4.04it/s]"
-     ]
-    }
-   ],
-   "source": [
-    "for i in trange(5000):\n",
-    "    sess.run(train_step, sample_batch())\n",
+   "outputs": [],
+   "source": [
+    "for i in trange(15000):\n",
+    "    memory = list(pool.prev_memory_states)\n",
+    "    rollout_obs, rollout_actions, rollout_rewards, rollout_mask = pool.interact(10)\n",
+    "    train_on_rollout(rollout_obs, rollout_actions,\n",
+    "                     rollout_rewards, rollout_mask, memory)\n",
     "\n",
     "    if i % 100 == 0:\n",
     "        rewards_history.append(np.mean(evaluate(agent, env, n_games=1)))\n",
     "        clear_output(True)\n",
-    "        plt.plot(rewards_history, label='rewards')\n",
+    "        plt.plot(rewards_history, label='rewards', linewidth=3)\n",
```

Comment:
Are the increased `linewidth` and `fontsize` really necessary?

Reply:
It's not necessary, it just looks better IMHO. It's sometimes more difficult to look at thin lines if one is distant from a display.

Reply:
I don't insist btw